lib/display/display.js

```diff
@@ -75,49 +78,61 @@ export default class Display {
     this.computeScale()
   }
 
-  getDefaultZoomFactors (data) {
-    return [{
-      minScale: 0,
-      gridCellSize: 25,
-      internalVertexFactor: 1000000,
-      angleConstraint: 45,
-      mergeVertexThreshold: 200
-    }, {
-      minScale: 1.5,
-      gridCellSize: 0,
-      internalVertexFactor: 0,
-      angleConstraint: 5,
-      mergeVertexThreshold: 0
-    }]
-  }
-
-  updateActiveZoomFactors (scale) {
-    var updated = false
-    for (var i = 0; i < this.zoomFactors.length; i++) {
-      var min = this.zoomFactors[i].minScale
-      var max = (i < this.zoomFactors.length - 1)
-        ? this.zoomFactors[i + 1].minScale
-        : Number.MAX_VALUE
+  getDefaultZoomFactors(data) {
+    return [
+      {
+        angleConstraint: 45,
+        gridCellSize: 25,
+        internalVertexFactor: 1000000,
+        mergeVertexThreshold: 200,
+        minScale: 0
+      },
+      {
+        angleConstraint: 5,
+        gridCellSize: 0,
+        internalVertexFactor: 0,
+        mergeVertexThreshold: 0,
+        minScale: 1.5
+      }
+    ]
+  }
+
+  updateActiveZoomFactors(scale) {
+    let updated = false
+    console.log('scale: ', this.scale)
```

Comment:
Is this called often? If it's intentional it's good but just want to double check it doesn't affect performance

Reply:
whoops, that's debugging code. I got rid of it in https://github.com/conveyal/transitive.js/pull/72/commits/10ab23c610d5990fb1f605992091025d33e60cc1